ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.17.yaml

```
@@ -93,18 +97,21 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-byo-iam-profile
-- as: aws-ipi-byo-iam-role-arm-f60
+- as: aws-ipi-byo-iam-role-efs-arm-f60-custom-cert
```

@jianlinliu Hmm, needs a bit info, is this `-f60` frequency not as a suffix expected? The same in https://github.com/openshift/release/pull/56160/files#diff-fd088e7048faa35212d70b16f01645b6f15276135dabfd8af796458bf88a4179R65 . It is not consistent with 4.15 PR ->
https://github.com/openshift/release/pull/56126/files#diff-6c9d8f80cbd2256599a522458fa761a0aa949af8c99c8b7139b6456005d0ee11R67 .
Seems should be ->
```suggestion
- as: aws-ipi-byo-iam-role-efs-arm-custom-cert-f60
```
right?

---

`custom-cert` is added as a test chain, for test chain, we usually append it after frequency field, the same example - `destructive`

---

Got it, thanks for the double confirm!